src/processor/operator/partitioner.cpp

```diff
@@ -55,7 +54,7 @@ partition_idx_t PartitionerSharedState::getNextPartition(idx_t partitioningIdx,
     if (nextPartitionIdxToReturn >= numPartitions[partitioningIdx]) {
         return INVALID_PARTITION_IDX;
     }
-    progressSharedState.partitionsDone++;
+    ++progressSharedState.partitionsDone;
```

Comment:
Does this change actually matter?

Reply:
Not really. It's more of a code style change (pre-increment vs. post-increment), which I forgot to revert. Will clean it up.